minorminer/utils/parallel_embeddings.py

```diff
@@ -197,13 +199,33 @@ def find_multiple_embeddings(
     return embs
 
 
+def lattice_size(T: nx.Graph = None) -> int:
```

Comment:
This signature allows for a parameterless call, which fails with an `AttributeError` exception.